app/controllers/finish_installation_controller.rb

```diff
@@ -10,6 +10,9 @@ class FinishInstallationController < ApplicationController
   before_action :ensure_no_admins, except: %w[confirm_email resend_email]
 
   def index
+    @setting_up_discourse_id = ENV["DISCOURSE_SKIP_EMAIL_SETUP"] == "1"
```

Comment:
I wonder if we should use "DISCOURSE_WITH_DISCOURSE_ID" (or something along those lines) for a "better" name?